### ISSUE TITLE
Do not allow `TypeVarTuple` after `TypeVar` with a default

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#3265]
There's a pattern that is allowed from typing perspective:

```python
class X[T=int, *Ts=*tuple[int, ...]]: ...
```

https://mypy-play.net/?mypy=latest&python=3.13&gist=cbd582efc127e91a38a022d99ea7269f

But, it is a really bad pattern, because you cannot specify `*Ts` without specifing `T`

So, we need to find `type_params` definitions where `TypeVar` with default is followed by `TypeVarTuple` with default.

**Comment by Tapeline** [comment]
Hi. I want to address this issue. 

Did some research on it. I thinks the solution is as follows: 
1. Identify the typevars with defaults
2. On each class declaration check for a pattern

However, there are some questions:
1. Should we look only for TypeVar and TypeVarTuple identifiers or should we also consider possible `import as`?
2. Should we consider scopes, or pretend that every typevar is globally declared?

**Comment by sobolevn** [comment]
We should only check for pep695 definitions. No imports. No global variables.

**Comment by Tapeline** [comment]
Given an example:
```py
from typing import TypeVar

class MyClass[T]:
    V = TypeVar("V", default=int)
    
    def method(self, v: V) -> V:
        ...

    # V is accessible here

# but not here
```

Should we forget about V's existence after the class declaration, or pretend it's defined at the top level?

The first approach is more correct, but the second one is easier to implement

**Comment by Tapeline** [comment]
Where the rule should go? Into OOP or into Best practices section?

**Comment by sobolevn** [comment]
Best practices, near https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/violations/best_practices.py#L2969
